lean/src/test/aime_1995_p7.lean

```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2021 OpenAI. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kudzo Ahegbebu
+-/
+import data.real.basic
+import data.real.sqrt
+import analysis.special_functions.trigonometric
+
```

Comment:
there shall not be more than 1 empty line.... otherwise the linter of mathlib will complain